src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -216,9 +216,12 @@ CreateResourceRequestV2 createDatasetResourceRequest(
 
     CreateResourceRequestV2 req = new CreateResourceRequestV2().resourceId(datasetId.toString());
 
-    req.putPoliciesItem(
-        IamRole.ADMIN.toString(),
-        createAccessPolicyOne(IamRole.ADMIN, samConfig.adminsGroupEmail()));
+    Optional.of(samConfig.adminsGroupEmail())
```

Comment:
Using `Optional.of()` means that you know the argument is non-null. I think you want `Optional.ofNullable`.
```suggestion
    Optional.ofNullable(samConfig.adminsGroupEmail())
```